mocha.css

```diff
@@ -220,12 +220,16 @@ body {
   font-size: 12px;
   margin: 0;
   color: #888;
+  background-color: #FFF;
+  border: 1px solid #000;
+  box-shadow: 0 0 10px #333;
```

Comment:
This ensures that the report block always has a white color otherwise, the text can become unreadable